### ISSUE TITLE
:c 🌵 

### ISSUE THREAD

**Issue by AdamTeodoro** [angular/vscode-ng-language-service#1308]
# 🐞 bug report

### Is this a regression?

Not

### Description

The atualization have created an bug in the autocomplete, my autocomplete is'not work

If the bug is caused or experienced by a particular source code, please provide a minimal
reproduction of that source code.

## Bug Type
What does this bug affect

- [ X] Angular Language Service VSCode extension
- [ ] Angular Language Service server

## Reproduction

1. Install the extension ANGULAR LANGUAGE SERVICE IN THE LATEST VERSION
2. Go to PREFERENCES>SETTINGS>EXTENSIONS> ANGULAR LANGUAGE SERVICE
3. SELECT  THE BOX OPTION> EXPERIMENTAL-IVY
4. TEST THE TEMPLATE AUTOCOMPLETE CODE 

SORRY MY ENGLISH IS BAD

**Comment by atscott** [comment]
@AdamTeodoro Can you provide the logs by running the "Angular: Open Angular Server log" command?

**Comment by AdamTeodoro** [comment]
My logs

[](https://stackblitz.com/edit/angular-ivy-oxg4cu?file=src/app/myLogs.log)

My **tsconfig.json**
```

{
  "compileOnSave": false,
  "compilerOptions": {
    "baseUrl": "./",
    "outDir": "./dist/out-tsc",
    "sourceMap": true,
    "declaration": false,
    "downlevelIteration": true,
    "experimentalDecorators": true,
    "moduleResolution": "node",
    "importHelpers": true,
    "target": "es2015",
    "module": "es2020",
    "lib": [
      "es2018",
      "dom"
    ],

  },
  "angularCompilerOptions": {
    "fullTemplateTypeCheck": true,
    "preserveWhitespaces": false,
    "strictTemplates": true
  }
}
```

Other peoples with the problem:

[](https://stackoverflow.com/questions/65688608/angular-language-service-in-vscode-does-not-work-because-it-is-not-an-angular-pr)

**Comment by atscott** [comment]
@AdamTeodoro - The logs show that you do not have the experimental ivy language service enabled. Can you enable it in the settings?
![image](https://user-images.githubusercontent.com/479713/115903225-2f059f00-a418-11eb-8bf3-f507d7614dc2.png)

**Comment by AdamTeodoro** [comment]
logs updated: https://stackblitz.com/edit/angular-ivy-g1lff3?file=src/app/myLogs.log


The problem is also involved with the Experimental-ivy-prompt

**Comment by atscott** [comment]
@AdamTeodoro Thanks for the additional logs. Are you able to share an example project with this issue? We need to look at the actual TS code for the component of the template.

**Comment by AdamTeodoro** [comment]
I dont cant the share all code, but i can share one part of code with you, what code part you have preference?

**Comment by AdamTeodoro** [comment]
não, não entendi como, mas apareceu um espaço na div olha na foto com está ou estava no template:
![image](https://user-images.githubusercontent.com/66885644/116723840-47e0f800-a9b6-11eb-96ff-971e61654e57.png)

**Comment by atscott** [comment]
@AdamTeodoro - It looks like things are working in your screenshot. Are you able to create a small example project where you can reproduce this issue? It doesn't have to be the project you are working on. If you create a new project, do you see the same issue?
